src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -386,7 +402,7 @@ private void FireProcessingQueueEmpty()
         {
             long blockProcessingTimeInMicrosecs = _stopwatch.ElapsedMicroseconds();
             Metrics.LastBlockProcessingTimeInMs = blockProcessingTimeInMicrosecs / 1000;
-            Metrics.RecoveryQueueSize = _recoveryQueue.Reader.Count;
+            Metrics.RecoveryQueueSize = _queueCount;
```

Comment:
I don't think this is correct. `_queueCount` is for the whole queue, so `Recovery+Processing+(isProcessingOneNow ? 1 : 0)`

Reply:
Is still wrong, at least at time when logs as processing decrements; will do follow up fix

recover     1 | process     0